analyzers/tests/SonarAnalyzer.UnitTest/Rules/DisablingRequestValidationTest.cs

```diff
@@ -0,0 +1,69 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2021 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarAnalyzer.Common;
+using SonarAnalyzer.UnitTest.MetadataReferences;
+using SonarAnalyzer.UnitTest.TestFramework;
+using CS = SonarAnalyzer.Rules.CSharp;
+using VB = SonarAnalyzer.Rules.VisualBasic;
+
+namespace SonarAnalyzer.UnitTest.Rules
+{
+    [TestClass]
+    public class DisablingRequestValidationTest
+    {
+        [DataTestMethod]
+        [DataRow("3.0.20105.1")]
+        [DataRow(Constants.NuGetLatestVersion)]
+        [TestCategory("Rule")]
+        public void DisablingRequestValidation_CS(string aspNetMvcVersion) =>
+            Verifier.VerifyAnalyzer(@"TestCases\DisablingRequestValidation.cs",
+                new CS.DisablingRequestValidation(AnalyzerConfiguration.AlwaysEnabled),
+                additionalReferences: NuGetMetadataReference.MicrosoftAspNetMvc(aspNetMvcVersion));
+
+        [DataTestMethod]
+        [DataRow("3.0.20105.1")]
+        [DataRow(Constants.NuGetLatestVersion)]
```

Comment:
I don't think we need the `DataRow` schema for purpose of this test. It won't change with different version so removing it could make it simple and save us few test runs (CSharp7, 7.1, 7.2, 7.3, 8, 9 for one of those nugets).

Reply:
I've originally meant this for the `_Disabled` tests. If you're aware of any behavior differences between 3.0.20105.1 and latest, we might keep it for the functional test.

Reply:
I realized now that 3.0.20105.1 is from 2012. I will just use 5.x which is the latest MVC version for .NET Framework, since 2014, and we shouldn't aim to support anything else (as .NET Framework is in maintenance mode).